test/extensions/EXT/SPV_EXT_shader_atomic_float_/atomicrmw_fsub_double.ll

```diff
@@ -0,0 +1,34 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv --spirv-ext=+SPV_EXT_shader_atomic_float_add %t.bc -o %t.spv
```

Comment:
Please also test -r here - what LLVM IR would be generated after the translation?

Reply:
Will add, thanks